fail2ban/tests/servertestcase.py

```diff
@@ -158,6 +158,9 @@ class Transmitter(TransmitterBase):
 	def setUp(self):
 		self.server = TestServer()
 		super(Transmitter, self).setUp()
+		# so far isStarted only tested but not used otherwise
+		# and here we don't really .start server
+		self.assertFalse(self.server.isStarted())
```

Comment:
I think it will be enough to check isStarted once (see below in testStartFailedSockExists)...
Additionally I'm against asserts in setup of the test cases (executed in each test)... so I will outsource it in separate case...